src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+interface Props {
+  perPage: number;
+  total: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  perPage,
+  total,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesList: number[] = getNumbers(1, Math.ceil(total / perPage));
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={
+            currentPage === 1 ? undefined : () => onPageChange(currentPage - 1)
+          }
+        >
+          «
+        </a>
```

Comment:
The use of the character '«' directly in the markup can be unclear to some developers. Consider using an HTML entity '&laquo;' for better readability and to ensure consistent rendering across different browsers.

Reply:
not valid for task